<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewProfile</class>
 <widget class="QDialog" name="ViewProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#666;
color:white;
font-family:&quot;Poppins&quot;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>751</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;
background-color: rgba(32, 32, 32,.9);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>Player Information</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="username_l">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border:1px solid gray;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="playtime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border:1px solid gray;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playtime_l">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>Playtime</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="profile_description">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border:1px solid gray;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="game_information">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border:1px solid gray;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="game_l">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>Game</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="discord_tag">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border:1px solid gray;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rank_data">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border:1px solid gray;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rank_l">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>Rank</string>
    </property>
   </widget>
   <widget class="QLabel" name="discord_tag_l">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>Discord Tag</string>
    </property>
   </widget>
   <widget class="QLabel" name="email_l">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border:1px solid gray;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="profile_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>Profile Description</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
